css/home-page-redesign.styl

```diff
@@ -40,34 +42,45 @@ GRAY = #EFF2F5
     top: 4px
 
   .standard-button
-    background: none
-    border: 4px solid #FFFFFF
+    border: 2px solid
     border-radius: 6px
-    padding: 0.75em 1.5em
+    border-color: #FFFFFF
+    background: none
+    padding: 0.75em 1.75em
     transition: background 500ms ease
-    margin-top: 50px
+    margin: 50px 10px 0 10px
     font-size: 15px
+    font-weight: bold
+    text-decoration: none
 
     @media (max-width: 900px)
       margin-top: 20px
 
-    &:hover
-      background: rgba(0, 0, 0, 0.3)
+    &:visited
+      color: #FFFFFF
+
+    &:hover,
+    &:active,
+    &:focus
+      background: #FFFFFF
+      color: DARK-BLUE-GREEN
 
   .home-link-down
+    position: absolute
+    bottom: 0
     width: 100%
     text-align: center
     border: none
-    background-color: TEAL
+    background: none
     color: #FFFFFF
     font-size: 50px
     line-height: 50px
 
-  .home-intro
-    color: #FFFFFF
+    @media (max-width: 900px)
+      display: none
 
-    h1
-      color: #FFFFFF
+  .home-intro
+    background-color: rgba(91, 189, 193, 0.3)
```

Comment:
We really don't want some sort of colored overlay do we?  Probably should remove this.